tests/models/donut/test_processing_donut.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# Copyright 2022 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from transformers import DonutProcessor
```

Comment:
Should the `is_vision_available` method be checked first?
```
if is_vision_available():
    from transformers import DonutProcessor
```

Reply:
No, the processor itself doesn't rely on a framework, only `DonutFeatureExtractor` (soon to be called `DonutImageProcessor` - we're gonna deprecate the term "feature extractor") does, as it relies on Pillow.